rasa_nlu/data_router.py

```diff
@@ -234,3 +257,63 @@ def training_errback(failure):
         result.addErrback(training_errback)
 
         return result
+
+    def evaluate(self, data, parameters):
```

Comment:
I'd like to make the parameters explicit.

`def evaluate(self, data, project=RasaNLUConfig.DEFAULT_PROJECT_NAME, model=None):`